src/libexpr/lexer/prologue.inc

```diff
@@ -0,0 +1,93 @@
+#ifdef __clang__
```

Comment:
We should really avoid custom extensions like `.inc`, because that breaks syntax highlighting etc. in most editors.

Reply:
>> Can we give it a different extension?

> We should really avoid custom extensions like .inc, because that breaks syntax highlighting etc. in most editors.

Would you like to kindly suggest which extension should I use :) ?